devops/kubernetes/query-node/processorDeployment.ts

```diff
@@ -175,6 +203,8 @@ export class ProcessorServiceDeployment extends pulumi.ComponentResource {
                     },
                   ],
                   command: ['/bin/sh', '-c'],
+                  // Should we really use "-e ../.env" args could there be some env vars we have not
+                  // correctly overridden for this container?
                   args: ['cd query-node && yarn hydra-processor run -e ../.env'],
```

Comment:
We can replace this with what we have in our docker-compose

```suggestion
                  args: ['workspace', 'query-node-root', 'processor:start'],
```

Reply:
done in 

86f57bd